### DISCUSSION TITLE
v1.0.0

### DISCUSSION THREAD

**Discussion by MrKrishnaAgarwal** [MrKrishnaAgarwal/Git-CheatSheet#9] [Announcements]
# Hello, Everyone 👋🏻 

Welcome to our first release of this open source project.

## What is this?

This is a git cheatsheet with almost all commands and with a few resources to learn more about git.

## How to use this?

You can use this cheatsheet to learn git or to refresh your memory about git commands.

## How to contribute?

You can contribute to this project by adding more commands or by adding more resources to learn more about git.

## How to contact us?

You can contact us by creating an issue or by sending an email to [Krishna](mailto:dmkrishna.agarwal@gmail.com)

## License

This project is licensed under the MIT License - see the [LICENSE](/LICENSE) file for details

## What's Changed
* Git Commands by @MrKrishnaAgarwal in https://github.com/MrKrishnaAgarwal/Git-CheatSheet/pull/1
* Adding some more commands and resources by @arefathi in https://github.com/MrKrishnaAgarwal/Git-CheatSheet/pull/5
* Add setting up info commands by @swatiaggrawal in https://github.com/MrKrishnaAgarwal/Git-CheatSheet/pull/2
* Feature/resetbranch by @Huluvu424242 in https://github.com/MrKrishnaAgarwal/Git-CheatSheet/pull/6
* add new commands by @Huluvu424242 in https://github.com/MrKrishnaAgarwal/Git-CheatSheet/pull/7
* Rewrite Readme: adding, improving etc... by @MrKrishnaAgarwal in https://github.com/MrKrishnaAgarwal/Git-CheatSheet/pull/8

## New Contributors
* @MrKrishnaAgarwal made their first contribution in https://github.com/MrKrishnaAgarwal/Git-CheatSheet/pull/1
* @arefathi made their first contribution in https://github.com/MrKrishnaAgarwal/Git-CheatSheet/pull/5
* @swatiaggrawal made their first contribution in https://github.com/MrKrishnaAgarwal/Git-CheatSheet/pull/2
* @Huluvu424242 made their first contribution in https://github.com/MrKrishnaAgarwal/Git-CheatSheet/pull/6

**Full Changelog**: https://github.com/MrKrishnaAgarwal/Git-CheatSheet/commits/v1.0.0

**Note: This release includes all the PR's made till date. i.e. 20th Nov 2022**

<hr /><em>This discussion was created from the release <a href='https://github.com/MrKrishnaAgarwal/Git-CheatSheet/releases/tag/v1.0.0'>v1.0.0</a>.</em>

**Comment by MrKrishnaAgarwal** [comment]
@arefathi, @Huluvu424242 and @swatiaggrawal  
Request you to add more resources and commands to this cheatsheet, this milestone wouldn't be possible without you.
I was also constantly improving this repository, but we can't do it alone.
So let's collaborate and work on it

**Reply by swatiaggrawal** [reply]
Sure will work on it! 
@MrKrishnaAgarwal

**Reply by MrKrishnaAgarwal** [reply]
thank you!

**Reply by Huluvu424242** [reply]
Sorry i have overlooked this posting.
Congratulation to the 1.0.0 release. :tada:
In the future i would like to contibute again. 
(currently i try to solve https://adventofcode.com/ puzzles :grinning:)

**Reply by MrKrishnaAgarwal** [reply]
That's great!
Thanks a lot @Huluvu424242...

**Comment by swatiaggrawal** [comment]
It was a great experience contributing for the first time. 😁
@MrKrishnaAgarwal

**Reply by MrKrishnaAgarwal** [reply]
Thankyou, it was a great experience to collaborate with you all too 🌟
